interface/VideoEncoderDefs.h

```diff
@@ -256,6 +256,7 @@ typedef struct VideoParamsCommon {
     uint32_t numRefFrames;
     VideoRateControl rcMode;
     VideoRateControlParams rcParams;
+    uint32_t referenceMode;
```

Comment:
It is a good observation, we would need to provide a better way of isolating parameters per codec to expose them.  If it makes sense we can create an issue to fix this for all cases.